examples/react/react-app/src/dojo/generated/setup.ts

```diff
@@ -29,7 +29,11 @@ export async function setup({ ...config }: Config) {
     await getSyncEntities(toriiClient, contractComponents as any);
 
     const client = await setupWorld(
-        new DojoProvider(config.manifest, config.rpcUrl)
+        new DojoProvider(
+            config.worldAddress,
```

Comment:
this shouldn't exist - only the manifest is passed now, have you re built everything?

Reply:
@ponderingdemocritus Yes, I've rebuilt everything from scratch. Also tried once more after seeing your comment and I see that DojoProvider expects world address as first argument.

I used the same syntax in dojo jam app as well. 
https://github.com/satyambnsal/castle-hexapolis/blob/main/client/src/dojo/setupNetwork.ts#L27

Reply:
I see 
when I see the DojoProvider constructor, it expects only manifest and url
```
constructor(manifest?: any, url: string = LOCAL_KATANA) {
        super(manifest.world.address);
        this.provider = new RpcProvider({
            nodeUrl: url,
        });

        this.contract = new Contract(
            manifest.world.abi,
            this.getWorldAddress(),
            this.provider
        );
        this.manifest = manifest;
    }
    ```
    Not sure what's wrong. exploring it bit more.

Reply:
Not sure what was the issue but after deleting node modules from react app as well as from top-level and rebuilding everything resolves the issue!.